### ISSUE TITLE
Pomoc

### ISSUE THREAD

**Issue by Tykeppaj** [JosefKuchar/bakalari-api#8]
Zdravím, 
rád bych poprosil o pomoc s tím, jak zobrazit XML, které mi má stránka vrátit po tom, co navštívím adresu např. při vypočítávání tokenu. Zkoušel jsem pár věcí, ale jelikož se moc neorientuji v této oblasti, tak jsem naprosto bezradný...

Předem děkuji za jakoukoliv odpověď.

**Comment by mariansam** [comment]
Zdravím, záleží na OS. Na Linuxu doporučuji můj [xmlcurl](https://github.com/mariansam/xmlcurl) (prostě `curl` + pretty-print + syntax-highlighting). Na Windows nevím, můžeš si vytvořit něco vlastního. Ale vždy by mělo fungovat výslednou URL prostě vložit do prohlížeče.

**Comment by Tykeppaj** [comment]
Jak jsem již psal, v této oblasti se absolutně nevyznám, ale určitě plánuji si někdy v blízké budoucnosti rozšířit obzory právě o tenhle typ věcí. 
Kde tedy mohu najít to XML, když v mém prohlížeči (Firefoxu) vyhledám např. https://www.example.com/login.aspx?gethx=JanNovak k vygenerování tokenu? Když to vyhledám, tak se mi ukáže jen normální přihlášení do bakalářů a nikde nic?

**Comment by mariansam** [comment]
Jak jsem teď zjistil, stránka pro generování tokenu mi taky nefunguje. Pro to vyzkoušej [tohle](https://github.com/mariansam/bakalari-token-generator). Ostatní moduly fungují dobře. Pokud se ti nelíbí dělat to pomocí zadávání URl do prohlížeče, můžeš vyzkoušet [Postman](https://www.getpostman.com/), ale ten já nerad používám, protože to není svobodný sfoftware...

**Comment by JosefKuchar** [comment]
@mariansam Oprav si ten token generátor předtím než ho budeš takhle propagovat

**Comment by JosefKuchar** [comment]
Teď jsem to zkusil a normálně mi to funguje
![image](https://user-images.githubusercontent.com/12010072/45371645-5999e500-b5eb-11e8-9b6c-a4b1daf9d6b2.png)

**Comment by mariansam** [comment]
@JosefKuchar Promiň, že ho takhle propaguju, je to ode mě blbý.. mně funguje, co ti to hází?
![image](https://user-images.githubusercontent.com/33497067/45371987-260b8a80-b5ec-11e8-9884-01037aab3093.png)

**Comment by mariansam** [comment]
Jinak jo, mně už to funguje taky, když jsem to zkoušel poprvé, měl jsem tam `/next/login.aspx`. Mělo by to jít.

**Comment by JosefKuchar** [comment]
Je to popsané v tom issue, které jsem otevřel v tvém repu

**Comment by vitkolos** [comment]
Jakože, abych to upřesnil – moc nerozumím v čem je problém (čeho se přesně týká toto issue), každopádně pro větší názornost sem mohu pullnout výpočet tokenu pomocí PHP (třeba do rootu)... Jakože kvalita mého kódu nic moc, ale pro pochopení to asi bude stačit. Nebo tkví problém někde jinde?

**Comment by mariansam** [comment]
Já jsem to pochopil tak, že @Tykeppaj neví, jak posílat HTTP requesty.

**Comment by vitkolos** [comment]
A nestačí to napsat do adresního řádku? Nebo o co jde? 😄 V PHP použije něco, v Pythonu něco jiného, na Node.js pravděpodobně taky jinak... Kde vězí problém? 😀 Obecně pro requesty doporučuji [Insomnii](https://insomnia.rest/). 😉

**Comment by mariansam** [comment]
Ano, napsat to do adresního řádku stačí, jenže jemu to nefungovalo. Asi tam měl `/next/login.aspx`.

**Comment by vitkolos** [comment]
Tak sem @Tykeppaj pošli tu adresu – když tam máš přihlašovací obrazovku, něco je špatně. 😉
